apps/angular-mongodb-e2e/fixtures/editTutorials.test.js

```diff
@@ -0,0 +1,104 @@
+import {URLS} from '../utils/route.util';
```

Comment:
rename file apps/angular-mongodb-e2e/fixtures/editTutorials.test.js to 
apps/angular-mongodb-e2e/fixtures/edit-tutorials.test.js